### ISSUE TITLE
quick question

### ISSUE THREAD

**Issue by yeahme49** [maykar/plex_assistant#89]
I'm working on adding music support and had a question. in helpers.py random is set, you randomize the media. I'm struggling to get this to work with music, but if I use media = pa.server.createPlayQueue(media, shuffle=1) in __init__.py Plex shuffles it and it works fine.

My question is there a reason for randomizing media with shuffle functions as opposed to having plex do it when creating the play queue? I plan to make a PR for my music support, just want to see the reasoning for this. Thanks

**Comment by maykar** [comment]
I actually added random as an option way at the start of the rewrite and playQueues weren't an option at the time. Since then, I added some things to Pychromecast for playQueues and jjlawren was kind enough to implement changes to both PlexAPI and the HA Plex integration so that playQueues are now a viable option.

TL;DR - There is no need for using random over shuffle, in fact shuffling the playQueue is preferable and I just didn't think of it since random was already there.

**Comment by maykar** [comment]
Also, thanks for taking this on, this was next on my list after the 1.0.0 rewrite, but I still have zero music on my Plex server. Look forward to seeing what you've got, feel free to continue the conversation on here or reach out to me on HA's discord/forums. Let me know if you have any more questions, watching your branch now 👍

**Comment by maykar** [comment]
Heads up:

[__init__.py](https://github.com/maykar/plex_assistant/commit/79debfbdee318c45b6a0e329517102c37033aba1)
[helpers.py](https://github.com/maykar/plex_assistant/commit/d47e96cd8c9fe29ddbdfff8bcefaa058577cc118)
